merp_reservation_strategy/models/stock_location.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 VentorTech OU
+# Part of Ventor modules. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class Location(models.Model):
+    _inherit = "stock.location"
+
+    @api.onchange('location_id')
+    def _onchange_parent_location(self):
+        self.removal_prio = self.location_id.removal_prio
```

Comment:
Hm... Please, clarify why you need this?

Reply:
It's a requirement.